Customization/T250/CodeSnippets/Step1.5.1/RS401000.aspx

```diff
@@ -45,7 +45,7 @@
           <%-- The added code --%>
           <RowTemplate>
             <px:PXSelector ID="edInvoiceNbr" runat="server"
-                DataField="InvoiceNbr" Enabled="False" AllowEdit="True" />
```

Comment:
You need to update all other code snippets with the same change. That is, in each RS401000.aspx snippet, we should not have this Enabled="False".
Same for other changes in this pull request.

Reply:
Yes. This change should be made everywhere else.